recipes/pygments-ansi-color/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "pygments-ansi-color" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/chriskuehl/pygments-ansi-color/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 23d51e6dd55bb7326206f033cdfad5adb8598d957028fdd7c6fc58e19bc696a0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - pygments
```

```suggestion
    - pygments !=2.7.3
```
https://github.com/chriskuehl/pygments-ansi-color/blob/f50788591cda6b1fa7c5b87666b62fc1190751f0/setup.py#L18